flink-runtime/src/test/java/org/apache/flink/runtime/rest/messages/job/savepoints/SavepointTriggerRequestBodyTest.java

```diff
@@ -44,8 +45,10 @@ public SavepointTriggerRequestBodyTest(
     public static Collection<Object[]> data() {
         return Arrays.asList(
                 new Object[][] {
-                    {new SavepointTriggerRequestBody("/tmp", true)},
-                    {new SavepointTriggerRequestBody("/tmp", false)}
+                    {new SavepointTriggerRequestBody("/tmp", true, null)},
+                    {new SavepointTriggerRequestBody("/tmp", false, null)},
+                    {new SavepointTriggerRequestBody("/tmp", true, new TriggerId())},
+                    {new SavepointTriggerRequestBody("/tmp", false, new TriggerId())},
```

Comment:
Is there a reason we have these tests for triggerSavepoint, but not stopWithSavepoint?

Reply:
No, it was probably just missed.